packages/patternfly-4/react-core/src/components/Popover/PopoverCloseButton.js

```diff
@@ -5,9 +5,9 @@ import { css } from '@patternfly/react-styles';
 import { Button } from '@patternfly/react-core';
 import { TimesIcon } from '@patternfly/react-icons';
 
-const PopoverCloseButton = ({ onClose }) => (
-  <div className={css(styles.popoverClose)}>
-    <Button onClick={onClose} variant="plain" aria-label="Action">
+const PopoverCloseButton = ({ onClose, ...rest }) => (
+  <div className={css(styles.popoverClose)} {...rest}>
+    <Button onClick={onClose} variant="plain" aria-label="Close">
```

Comment:
Maybe add the aria-label as prop  that can be applied to a button for localization purposes.

Reply:
Good point, I'll do that